### PR TITLE
Releasable library

### PR DESCRIPTION
**Pull request by ryanrdoherty**

### CODE REVIEW

**Comment by ryanrdoherty** [comment]
@Foxcapades @dmgaldi Nearly all these changes are to fix JavaDoc errors.  I had the build fail on JavaDoc errors (which are a lot more restrictive than they used to be), so had some work to do there.  Maybe just focus on the pom.xml changes?

Some of the contents of pom.xml were moved to [base-pom/pom.xml](https://github.com/VEuPathDB/base-pom), which you could also review.
The previous "poor-man's repo" was cleaned up and moved from the Dependencies directory to [here](https://github.com/VEuPathDB/maven-packages/tree/main/raw-packages).

File: Db/src/main/java/org/gusdb/fgputil/db/runner/SingleIntResultSetHandler.java [hunk 1]
```diff
@@ -3,6 +3,8 @@
 import java.sql.ResultSet;
 import java.sql.SQLException;
 
+import java.util.NoSuchElementException;
+
```
Comment (dmgaldi) [inline Db/src/main/java/org/gusdb/fgputil/db/runner/SingleIntResultSetHandler.java:7]: Is this needed if only referenced in javadoc?

**Review by dmgaldi — APPROVED** [review]

File: Db/src/main/java/org/gusdb/fgputil/db/runner/SingleIntResultSetHandler.java (hunk 1 above)
Comment (ryanrdoherty) [reply Db/src/main/java/org/gusdb/fgputil/db/runner/SingleIntResultSetHandler.java:7]: It's a mystery.  It used to build before but failed on Javadoc, so yes?  They are definitely checking for legit code references now; even methods fail in @link annotations.

File: Cache/src/main/java/org/gusdb/fgputil/cache/InMemoryCache.java [hunk 2]
```diff
@@ -14,9 +14,9 @@
  * Implements an efficient cache of objects mapped by key.  This class is
  * thread-safe and can be used in a static, application-wide way to cache
  * homogeneous objects.
- * 
- * @param S type of unique keys of values cached
- * @param T type of values cached
+ *
+ * Generic S type of unique keys of values cached
+ * Generic T type of values cached
```
Comment (Foxcapades) [inline Cache/src/main/java/org/gusdb/fgputil/cache/InMemoryCache.java:19]: These should still be doable as `@param` using `@param <S>` and `@param <T>`

**Review by Foxcapades — APPROVED** [review]

File: Cache/src/main/java/org/gusdb/fgputil/cache/InMemoryCache.java (hunk 2 above)
Comment (ryanrdoherty) [reply Cache/src/main/java/org/gusdb/fgputil/cache/InMemoryCache.java:19]: Thanks Ellie, I will fix.

**Pull request merged**